tests/e2e/api/controllers/_changes.spec.js

```diff
@@ -423,12 +436,89 @@ describe('changes handler', () => {
   });
 
   describe('Filtered replication', () => {
+    const bobsIds = [...DEFAULT_EXPECTED],
+          stevesIds = [...DEFAULT_EXPECTED],
+          couchVersionForBatching = '2.3.0';
+
+    let shouldBatchChangesRequests;
+
+    beforeAll(done => {
+      const options = {
+        hostname: constants.COUCH_HOST,
+        port: constants.COUCH_PORT,
+        auth: auth.user + ':' + auth.pass,
+        path: '/'
+      };
+
+      const req = http.request(options, res => {
+        let body = '';
+        res.on('data', data => body += data);
+        res.on('end', () => {
+          try {
+            body = JSON.parse(body);
+            shouldBatchChangesRequests = semver.lte(couchVersionForBatching, body.version);
+            done();
+          } catch (e) {
+            done(e);
+          }
+        });
+      });
+
+      req.on('error', e => done(e));
+      req.end();
+    });
+
     beforeEach(done => getCurrentSeq().then(done));
 
-    const bobsIds = [...DEFAULT_EXPECTED],
-          stevesIds = [...DEFAULT_EXPECTED];
+    it('should successfully fully replicate in batches', () => {
+      if (!shouldBatchChangesRequests) {
+        return;
+      }
```

Comment:
I think it'd be better to move the `if` block outside the test so the test doesn't get reported as passing if it's not even run, eg:

```js
if (shouldBatchChangesRequests) {
  it('should successfully fully replicate in batches', () => {
    ...
```

This applies to the tests below too...

Reply:
I've ended up doing it differently, cause the test-suite list is evaluated before before/after hooks (apparently), so you can't change which tests run while running tests. 

So I merged 2 tests into 1 and conditionally run assertions on the result set.

Reply:
Ahh ok. Thanks.